src/js/lib/emailsignupURL.js

```diff
@@ -0,0 +1,3 @@
+export default function emailsignupURL(listId) {
+    return  window.location.origin === 'https://www.theguardian.com' ? 'https://www.theguardian.com/email/form/plaindark/' + listId : 'https://m.code.dev-theguardian.com/email/form/plaindark/' + listId;
```

Comment:
we can use [template strings](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) here - ❤️ ES6